nsxt/policy_utils.go

```diff
@@ -382,6 +382,18 @@ func nsxtPolicyPathResourceImporter(d *schema.ResourceData, m interface{}) ([]*s
 	return rd, nil
 }
 
+func nsxtVPCPathResourceImporter(d *schema.ResourceData, m interface{}) ([]*schema.ResourceData, error) {
+	rd, err := nsxtPolicyPathResourceImporterHelper(d, m)
+	if err != nil {
+		return rd, err
+	}
+	projectID, vpcID := getContextDataFromSchema(d)
```

Comment:
don't we need something like this for all project-scoped resources?

Reply:
If you refer to the importer, then it's more complicated with project/multitenancy resources, as we use the same resource in both project and non-project contexts while VPC resources are VPC-only.
Also as with legacy we used to import resources by id (instead of by path as we currently do), so that import-id could be an NSX id and not a path. We don't have this case with VPC either.